docs/install/kubernetes.md

```diff
@@ -0,0 +1,152 @@
+   ```
+
+1. Create a `values.yaml` with the configuration settings you'd like for your
+   deployment. For example:
+
+   ```yaml
+   coder:
+     # You can specify any environment variables you'd like to pass to Coder
+     # here. Coder consumes environment variables listed in
+     # `coder server --help`, and these environment variables are also passed
+     # to the workspace provisioner (so you can consume them in your Terraform
+     # templates for auth keys etc.).
+     #
+     # Please keep in mind that you should not set `CODER_ADDRESS`,
+     # `CODER_TLS_ENABLE`, `CODER_TLS_CERT_FILE` or `CODER_TLS_KEY_FILE` as
+     # they are already set by the Helm chart and will cause conflicts.
+     env:
+       - name: CODER_ACCESS_URL
+         value: "https://coder.example.com"
+       - name: CODER_PG_CONNECTION_URL
+         valueFrom:
+           secretKeyRef:
+             # You'll need to create a secret called coder-db-url with your
+             # Postgres connection URL like:
+             # postgres://coder:password@postgres:5432/coder?sslmode=disable
+             name: coder-db-url
+             key: url
+
+       # This env variable controls whether or not to auto-import the
+       # "kubernetes" template on first startup. This will not work unless
+       # coder.serviceAccount.workspacePerms is true.
+       - name: CODER_TEMPLATE_AUTOIMPORT
+         value: "kubernetes"
+
+     #tls:
+     #  secretName: my-tls-secret-name
+   ```
+
+   > You can view our
+   > [Helm README](https://github.com/coder/coder/blob/main/helm#readme) for
+   > details on the values that are available, or you can view the
+   > [values.yaml](https://github.com/coder/coder/blob/main/helm/values.yaml)
+   > file directly.
+
+1. Run the following commands to install the chart in your cluster.
+
+   ```sh
+   helm install coder ./coder_helm_x.y.z.tgz \
+       --namespace coder \
+       --values values.yaml
+   ```
+
+You can watch Coder start up by running `kubectl get pods`. Once Coder has
+started, the `coder-*` pods should enter the `Running` state.
+
+You can view Coder's logs by getting the pod name from `kubectl get pods` and
+then running `kubectl logs <pod name>`. You can also view these logs in your
+Cloud's log management system if you are using managed Kubernetes.
+
+## Upgrading Coder via Helm
+
+To upgrade Coder in the future or change values,
+you can run the following command with a new `coder_helm_x.y.z.tgz` file from GitHub releases:
+
+```console
+$ helm upgrade coder ./coder_helm_x.y.z.tgz \
+    --namespace coder \
+    -f values.yaml
+```
+
+## Troubleshooting
+
+### Kubernetes-based workspace is stuck in "Connecting..."
+
+Ensure you have an externally-reachable `CODER_ACCESS_URL` set in your helm chart. If you do not have a domain set up,
+this should be the IP address of Coder's LoadBalancer (`kubectl get svc -n coder`).
+
+See [troubleshooting templates](../templates.md#creating-and-troubleshooting-templates) for more steps.
```

Comment:
Added some troubleshooting steps here

Reply:
I was experimenting with this today. So you need to run `helm upgrade` after retrieving the IP address from the initial `helm install`. You may want to mention that.

Reply:
added this 👍🏼